src/javascript/app/ng-wig/ng-wig.component.js

```diff
@@ -51,7 +51,10 @@ angular.module('ngWig')
 
       this.$onInit = () => {
         let placeholder = Boolean(this.placeholder);
-        this.ngModelController.$render = () =>  !placeholder ? $container.html(this.ngModelController.$viewValue || '<p></p>') : $container.empty();
+        this.ngModelController.$render = () => this.ngModelController && this.ngModelController.$viewValue
+          ? $container.html(this.ngModelController.$viewValue)
+          : placeholder ? $container.empty()
```

Comment:
I would suggest to write it in separate if statements because it is very confusing to read.